bridge/contracts/libraries/errorCodes/BaseParserLibraryErrorCodes.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT-open-group
+pragma solidity ^0.8.11;
+
+library BaseParserLibraryErrorCodes {
+    // BaseParserLibrary error codes
+    uint16 public constant BASEPARSERLIB_OFFSET_PARAMETER_OVERFLOW = 1000; // "BaseParserLibrary: An overflow happened with the offset parameter!"
```

Comment:
Does using a uint256 makes any difference in the gas cost of the reverted tx?

Reply:
And how about using `string` instead of a uint16, this would spare a type conversion. Would the type conversion be cheaper than storing the string directly?

Reply:
Trying an approach using bytes32 which will restrict the size of the numbers while keeping it able to be easily converted to `string` using `string(abi.encodePacked(errorCode))`